src/chat.rs

```diff
@@ -1450,6 +1466,18 @@ pub fn get_chat_msgs(
     flags: u32,
     marker1before: Option<MsgId>,
 ) -> Vec<MsgId> {
+    match hide_device_expired_messages(context) {
+        Err(err) => warn!(context, "Failed to delete expired messages: {}", err),
+        Ok(messages_deleted) => {
```

Comment:
There's probably not much in it, but I'd be tempted to match on `Ok(true)` instead of having a single if expression inside this match.

Reply:
I have tried it, but then it is less clear what does `true` mean in `Ok(true)`. Here I have a `messages_deleted` name, so it is obvious what `if messages_deleted` checks for.